OpenDreamShared/Compiler/DMPreprocessor/DMPreprocessor.cs

```diff
@@ -154,6 +154,13 @@ public class DMPreprocessor {
 
                         break;
                     }
+                    case TokenType.DM_Preproc_If:
+                    {
+                        //TODO Implement #if properly
+                        SkipIfBody();
+                        _currentLine.Add(new Token(TokenType.Warning, token.Text, token.SourceFile, token.Line, token.Column, "#if is not implemented"));
```

Comment:
This won't output the warning token if the line is otherwise empty
```suggestion
                        _result.Add(new Token(TokenType.Warning, token.Text, token.SourceFile, token.Line, token.Column, "#if is not implemented"));
```

Reply:
This PR is blocked until this no longer causes an EOF error.

Reply:
As of 8e2523293d6ee8c8f90f967059eb69161a63c17b, this can be merged after the above suggestion.